elements/pfe-sass/maps/_colors.scss

```diff
@@ -22,7 +22,7 @@ $pfe-colors: (
 
   text--on-saturated:          $pf-color-white,
   text--muted--on-saturated:   $pf-color-black-300,
-  link--on-saturated:          $pf-color-white,
+  link--on-saturated:          $pf-color-black-300,
```

Comment:
You've changed the `link--on-saturated` to gray but I think you meant to change the `link--visited--on-saturated`